### ISSUE TITLE
`nvm use` version behavior is different from `nvm install`

### ISSUE THREAD

**Issue by FeodorFitsner** [coreybutler/nvm-windows#691]
`nvm install 16` installs the latest 16.x release:

```
Downloading node.js version 16.13.0 (64-bit)... 
Complete
Installation complete. If you want to use this version, type
nvm use 16.13.0
```

while `nvm use 16` is trying to use `16.0.0`:

```
node v16.0.0 (64-bit) is not installed.
```

**Comment by coreybutler** [comment]
which version of NVM4W are you running?

**Comment by FeodorFitsner** [comment]
1.1.8. [install](https://github.com/coreybutler/nvm-windows/blob/master/src/nvm.go#L224-L228) and [use](https://github.com/coreybutler/nvm-windows/blob/master/src/nvm.go#L398) functions treat input version differently.

**Comment by coreybutler** [comment]
This is now fixed in the master branch. It won't be released until 1.1.9, which is currently held up until our new code signing certificate is delivered (in progress, thanks to a sponsor).

**Comment by FeodorFitsner** [comment]
Cool, thank you!

**Comment by 0liver** [comment]
@coreybutler We're eagerly waiting for the signed release v1.1.9 😁 I'm glad to hear someone took on the sponsorship for the certificate.

**Comment by coreybutler** [comment]
Resolved in 5751e53159feeb7248badbe4f3d36f24c6762a0a. v1.1.9 will be release by or on Dec 10, 2021.

**Comment by 0liver** [comment]
Thank you, @ajyong!
---
![image](https://user-images.githubusercontent.com/164360/145792455-2393542b-c81a-480d-9984-f32e2d9ea29a.png)
